deployments/main.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 func main() {
-	envs := []string{"generic", "EKS", "GKE", "docker", "minikube", "microk8s", "k3s"}
+	envs := []string{"generic", "EKS", "GKE", "docker", "minikube", "microk8s", "k3s", "AKS"}
```

Comment:
can you make the order like {generic, docker, minikube, microk8s, k3s, gke, eks, aks}?